modules/nw-route-specific-annotations.adoc

```diff
@@ -16,55 +16,122 @@ To create a whitelist with multiple source IPs or subnets, use a space-delimited
 //*route definition* for the route to alter its configuration.
 
 .Route annotations
-[cols="3*", options="header"]
+[cols="4*", options="header"]
 |===
-|Variable | Description | Environment variable used as default
-|`haproxy.router.openshift.io/balance`| Sets the load-balancing algorithm. Available options are `random`, `source`, `roundrobin`, and `leastconn`.  The default value is `source` for TLS passthrough routes. For all other routes, the default is `random`. |`ROUTER_TCP_BALANCE_SCHEME` for passthrough routes. Otherwise, use `ROUTER_LOAD_BALANCE_ALGORITHM`.
-|`haproxy.router.openshift.io/disable_cookies`| Disables the use of cookies to track related connections. If set to `'true'` or `'TRUE'`, the balance algorithm is used to choose which back-end serves connections for each incoming HTTP request. |
-|`router.openshift.io/cookie_name`| Specifies an optional cookie to use for
+|Variable 
+|Description 
+|Ingress controller field
+|Ingress controller annotation
```

Comment:
As I suggested for "Ingress controller field", can you delete this column and incorporate the information in this column (for the one row that uses it) into the "Description" column?  

I cannot even see this column without scrolling, and so I didn't even know the table had this column until I looked at the AsciiDoc.  I am currently viewing the preview on a very large display, so I expect that most people who look at this table will not see the rightmost column and will not realize it exists.

Reply:
Ack.

Reply:
My personal opinion is that keeping it as "Description" is better because it is shorter and because most rows don't mention an IngressController field, but maybe "Description & Ingress Controller Field" improves discoverability.